linkcheck/logger/sitemapxml.py

```diff
@@ -81,7 +82,11 @@ def log_filter_url(self, url_data, do_print):
         # initialize prefix and priority
         if self.prefix is None:
             if not url_data.url.startswith(HTTP_SCHEMES):
```

Comment:
Hey so there was that `not startswith(('http:', 'https:'))` suggestion I made earlier somewhere (in the proxy code?), could it use HTTP_SCHEMES as well?